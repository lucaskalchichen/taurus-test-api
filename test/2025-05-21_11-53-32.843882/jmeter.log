2025-05-21 11:53:36,772 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:53:36,783 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:53:36,784 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:53:36,784 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882/jmeter-bzt.properties
2025-05-21 11:53:36,785 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882/system.properties
2025-05-21 11:53:36,788 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:53:36,788 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:53:36,788 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:53:36,788 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:53:36,788 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:53:36,789 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:53:36,792 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:53:36,792 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:53:36,792 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:53:36,792 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:53:36,792 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:53:36,798 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:53:36,798 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882
2025-05-21 11:53:36,798 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882
2025-05-21 11:53:36,800 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882
2025-05-21 11:53:36,805 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:53:36,806 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882'
2025-05-21 11:53:36,878 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:53:36,889 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:53:36,891 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:53:36,893 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882/modified_requests.jmx
2025-05-21 11:53:36,910 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:53:36,911 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:53:36,918 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:53:36,936 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:53:36,937 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:53:36,937 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:53:36,943 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:53:36,943 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:53:37,078 INFO o.a.j.JMeter: Running test (1747839217078)
2025-05-21 11:53:37,087 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:53:37,087 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:53:37,087 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:53:37,087 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:53:37,093 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:53:37,105 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:53:37,105 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:53:37,105 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:53:37,107 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:53:37,108 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:53:37,108 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:53:37,108 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:53:37,108 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:53:37,161 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:53:37,230 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:53:37,302 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:53:37,374 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:53:37,445 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:53:37,514 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:53:37,585 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:53:37,656 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:53:37,727 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:53:37,798 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:53:37,869 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:53:37,940 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:53:38,011 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:53:38,083 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:53:38,153 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:53:38,223 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:53:38,294 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:53:38,366 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:53:38,436 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:53:38,508 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:53:38,579 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:53:38,649 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:53:38,721 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:53:38,792 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:53:38,863 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:53:38,934 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:53:39,005 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:53:39,076 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:53:39,146 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:53:39,218 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:53:39,288 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:53:39,359 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:53:39,431 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:53:39,501 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:53:39,573 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:53:39,644 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:53:39,715 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:53:39,786 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:53:39,856 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:53:39,928 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:53:39,999 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:53:44,436 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:53:44,436 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:53:50,292 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:53:50,292 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:53:54,342 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:53:54,343 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:53:56,305 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:53:56,305 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:53:56,305 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:53:56,306 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:53:56,990 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:53:56,990 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:53:59,979 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:53:59,979 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:54:01,409 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:54:01,409 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:54:04,911 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:54:04,912 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:54:04,912 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:54:04,912 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:54:09,074 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:54:09,074 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:54:09,991 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:54:09,991 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:54:10,657 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:54:10,658 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:54:12,597 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:54:12,597 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:54:14,095 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:54:14,095 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:54:14,756 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:54:14,757 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:54:15,946 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:54:15,946 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:54:18,324 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:54:18,324 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:54:18,324 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:54:18,324 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:54:19,004 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:54:19,004 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:54:20,342 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 11:54:20,342 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:54:20,781 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:54:20,781 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:54:22,372 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 11:54:22,372 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:54:24,053 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 11:54:24,054 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 11:54:24,054 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:54:24,054 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:54:25,880 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 11:54:25,880 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:54:27,228 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 11:54:27,250 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:54:27,333 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 11:54:27,333 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:54:28,122 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 11:54:28,122 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:54:28,952 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 11:54:28,953 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:54:30,431 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 11:54:30,431 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:54:30,519 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 11:54:30,520 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:54:31,422 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 11:54:31,422 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:54:32,125 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 11:54:32,125 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:54:33,022 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 11:54:33,022 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:54:33,909 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 11:54:33,909 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:54:35,247 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 11:54:35,248 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:54:35,357 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 11:54:35,357 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:54:36,388 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 11:54:36,389 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:54:37,709 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 11:54:37,709 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:54:38,384 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 11:54:38,385 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:54:38,393 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 11:54:38,395 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:54:38,395 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:54:38,397 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
