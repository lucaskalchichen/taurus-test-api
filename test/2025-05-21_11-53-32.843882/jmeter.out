Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-53-32.843882/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:53:36 ART (1747839216932)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 11:54:38 ART (1747839278396)
... end of run
